### PR TITLE
Support PFN CI

### PR DESCRIPTION
**Pull request by kmaehashi**
This depends on #36

### CODE REVIEW

**Comment by kmaehashi** [comment]
pfnCI, test this please.

**Comment by kmaehashi** [comment]
pfnCI, test this please.

**Comment by kmaehashi** [comment]
pfnCI, test this please.

**Comment by kmaehashi** [comment]
Rebased.

**Comment by kmaehashi** [comment]
pfnCI, test this please.

File: .pfnci/test.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -uex
+
+# Download NCCL
+gsutil -m cp -r gs://chainer-artifacts-pfn-public-ci/cupy-release-tools/nccl .
```
Comment (toslunar) [inline .pfnci/test.sh]: Could you comment how the data source will be maintained?

**Pull request merged**